mlflow/genai/judges/instructions_judge/__init__.py

```diff
@@ -395,11 +402,21 @@ def __call__(
             ChatMessage(role="user", content=user_content),
         ]
 
+        response_format = pydantic.create_model(
+            "ResponseFormat",
+            result=(
+                self._feedback_value_type or str,
+                pydantic.Field(description=self.description or "The result of the evaluation"),
+            ),
+            rationale=(str, pydantic.Field(description="The rationale for the evaluation")),
+        )
```

Comment:
@TomeHirata Anything we can do to make the `is_trace_based == True` case work with Databricks gpt-oss, which doesn't support structured outputs and tool calling together?